src/preferences/beatdetectionsettings.h

```diff
@@ -64,14 +64,14 @@ class BeatDetectionSettings {
             plugin_id);
     }
     QString getBeatPluginIdDefault() const {
-        return "qm-tempotracker";
+        return "qm-tempotracker:0";
```

Comment:
This redundant string should be replaced by `AnalyzerQueenMaryBeats::pluginInfo().id`